MAUI/PDF-Viewer/Save-a-Document.md

```diff
@@ -27,4 +27,30 @@ private void SaveDocument()
 {% endhighlight %}
 {% endtabs %}
 
-To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
\ No newline at end of file
+To save a document asynchronously, you may use the [SaveDocumentAsync](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.PdfViewer.SfPdfViewer.html#Syncfusion_Maui_PdfViewer_SfPdfViewer_SaveDocumentAsync_System_IO_Stream_System_Threading_CancellationToken_) method  of the `SfPdfViewer`.
+
+## Flatten annotation and formfield on save in .NET MAUI PDF Viewer
+
+Flattening refers to the process of converting interactive elements, such as annotations and form fields, into a static, uneditable format within a PDF document.The SfPdfViewer allows you to save a document to the specified file stream after flattened the specific annotations and formfields.
```

Comment:
The meaning varies. Please improve this content,
<img width="335" alt="image" src="https://github.com/syncfusion-content/maui-docs/assets/65169077/6eab3a00-2820-47cc-9908-a4c6aee76cf2">

Reply:
addressed the feedback